packages/types/src/clerk.ts

```diff
@@ -894,12 +894,14 @@ export type SignInProps = RoutingOptions & {
   /**
    * Enable experimental flags to gain access to new features. These flags are not guaranteed to be stable and may change drastically in between patch or minor versions.
    */
-  __experimental?: Record<string, any> & { newComponents?: boolean };
+  __experimental?: Record<string, any> & { newComponents?: boolean; combinedFlow?: boolean };
   /**
    * Full URL or path to for the waitlist process.
    * Used to fill the "Join waitlist" link in the SignUp component.
    */
   waitlistUrl?: string;
+
+  signUpProps?: SignUpProps;
```

Comment:
Move `signUpProps` to `__experimental` too?

Reply:
https://github.com/clerk/javascript/pull/4607/commits/da880cc14721f58b4041385b1a2804829239b6e2